indico/modules/events/editing/client/js/components/Timeline/judgment/UpdateFilesForm.jsx

```diff
@@ -26,20 +26,34 @@ import FinalTagInput from './TagInput';
 
 import './JudgmentBox.module.scss';
 
+const confirmOptions = [
+  {
+    value: 'confirm',
+    text: Translate.string('Confirm'),
+  },
+  {
+    value: 'confirm_and_approve',
+    text: Translate.string('Confirm & Approve'),
+  },
+];
+
 export default function UpdateFilesForm({setLoading}) {
+  const [confirmationType, setConfirmationType] = useState('confirm');
   const {fileTypes} = useSelector(selectors.getStaticData);
   const lastRevision = useSelector(selectors.getLastRevision);
   const staticData = useSelector(selectors.getStaticData);
   const {eventId, contributionId, editableType} = staticData;
   const dispatch = useDispatch();
   const files = getFilesFromRevision(fileTypes, lastRevision);
+  const option = confirmOptions.find(x => x.value === confirmationType);
```

Comment:
maybe do sth like

```javascript
const confirmOptions = {
  confirm: {
    value: 'confirm',
    text: Translate.string('Confirm'),
  },
  confirm_and_approve: {
    value: 'confirm_and_approve',
    text: Translate.string('Confirm & Approve'),
  },
};
```

this way you will not have to loop over the array just to get the object. Then you would also have to change `options={confirmOptions}` to `options={Object.values(confirmOptions)}`.

This is just a suggestion. Feel free to leave it like that

Reply:
I don't think this is needed at all. You can just check `confirmationType` directly below instead of `option.value`

Reply:
Yeah, true.